app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :users
+  has_many :comments
+  has_many :likes
```

Comment:
- [ ] The associations between models must be set up correctly according to the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png). Let's check [post model](https://github.com/Solobarine/whispers/blob/2fd9464bf2f5166c9438e3c51f580a5f1edc4a87/db/schema.rb#L36-L45) The same here, this means 🤔 The Post belongs-to user through the `'author_id'` column of the Post table. Now, we need to show that, the  `'author_id'` reference the User class modal.
Your code should look like this:

```ruby
  # app/models/post.rb
  belongs_to :author, class_name: 'User'
```